test/unit/puppetlabs/services/jruby/jruby_puppet_core_test.clj

```diff
@@ -167,10 +167,7 @@
                                     nil)]
 
       (testing "jruby-config default values are used if not provided"
-        ; jruby-utils defaults to jit when running 9k
-        (if jruby-schemas/using-jruby-9k?
-          (is (= :jit (:compile-mode initialized-jruby-config)))
-          (is (= :off (:compile-mode initialized-jruby-config))))
+        (is (= :jit (:compile-mode initialized-jruby-config)))
```

Comment:
Do we plan to update jruby-utils to remove the logic around selecting the default compile-mode? https://github.com/puppetlabs/jruby-utils/blob/6a77927e73c9fcade6265f8044242224565f1ace/src/clj/puppetlabs/services/jruby_pool_manager/jruby_core.clj#L24-L29

Reply:
Oh yeah, good call, that will need an update.

Reply:
I'll file a ticket about that.